dask_ml/_typing.py

```diff
@@ -0,0 +1,79 @@
+# Borrowed from https://github.com/pandas-dev/pandas/blob/master/pandas/_typing.py
```

Comment:
I'd prefer to take only what we need (e.g. ArrayLike).

STuff like pandas._libs, pandas.core is private to pandas, and so might break our CI without warning.

Reply:
Ok.  I will correct it.